src/cpp/StatisticsBackend.cpp

```diff
@@ -301,6 +302,9 @@ EntityId StatisticsBackend::init_monitor(
         CallbackMask callback_mask,
         DataKindMask data_mask)
 {
+    /* Deactivate statistics in case they were set */
+    unsetenv("FASTDDS_STATISTICS");
```

Comment:
Check that `unsetenv` works in windows

Reply:
Indeed, it does not exist on windows. Use `putenv("FASTDDS_STATISTICS=")` instead

Reply:
:+1: